webpack.config.js

```diff
@@ -0,0 +1,46 @@
+const webpack = require("webpack");
+
+module.exports = {
+	entry: {
```

Comment:
Just for future proofing, will it be easy to add our `postcss` stuff into this config, or it will have to be rewritten for an additional export or whatever?